docker/.env

```
@@ -336,4 +336,4 @@ DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=1
 THREAD_POOL_MAX_WORKERS=128
 
 #Option to disable login form for SSO
-DISABLE_PASSWORD_LOGIN=false
+DISABLE_PASSWORD_LOGIN=false
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)